app/src/components/market/sections/market_create/market_wizard_creator_container.tsx

```diff
@@ -26,6 +27,20 @@ const MarketWizardCreatorContainer: FC = () => {
   const [marketCreationStatus, setMarketCreationStatus] = useState<MarketCreationStatus>(MarketCreationStatus.ready())
   const [marketMakerAddress, setMarketMakerAddress] = useState<string | null>(null)
 
+  const getCompoundInterestRate = async (symbol: string): Promise<number> => {
+    const tokenSymbol = symbol.toLowerCase()
+    let cToken = `c${tokenSymbol}`
+    if (tokenSymbol === 'weth') {
+      cToken = 'ceth'
+    }
+    const compoundCollateralToken = cToken as KnownToken
+    const compoundTokenDetails = getToken(context.networkId, compoundCollateralToken)
+    const compoundService = new CompoundService(compoundTokenDetails.address, cToken, provider, account)
+    await compoundService.init()
+    const supplyRate = compoundService.calculateSupplyRateAPY()
+    return supplyRate
+  }
+
   const handleSubmit = async (marketData: MarketData, isScalar: boolean) => {
```

Comment:
Just checking - are we planning on adding compound support for scalar markets later on? Sounds good to me if so.

Reply:
Yes we do after this gets merged